tests/phpunit/tests/theme/wpThemeJson.php

```diff
@@ -3493,4 +3493,335 @@ function test_get_styles_for_block_with_content_width() {
 		$style_rules = $theme_json->get_styles_for_block( $metadata );
 		$this->assertSame( $expected, $root_rules . $style_rules );
 	}
+
+	/**
+	 * @dataProvider data_generate_spacing_scale_fixtures
+	 *
+	 * @ticket 56467
+	 *
+	 * Tests generating the spacing presets array based on the spacing scale provided.
+	 */
+	function test_set_spacing_sizes( $spacing_scale, $expected_output ) {
```

Comment:
Test method arguments should be documented at the end of the test method's docblock.

Reply:
done